pkg/backup/item_backupper.go

```diff
@@ -248,18 +266,18 @@ func (ib *defaultItemBackupper) backupItem(logger logrus.FieldLogger, obj runtim
 	return nil
 }
 
-func backupPodVolumes(log logrus.FieldLogger, backup *api.Backup, obj runtime.Unstructured, backupper restic.Backupper) (runtime.Unstructured, []error) {
-	if backupper == nil {
-		log.Warn("No restic backupper, not backing up pod's volumes")
+func (ib *defaultItemBackupper) backupPodVolumes(log logrus.FieldLogger, obj runtime.Unstructured, pod *corev1api.Pod, volumes []string) (runtime.Unstructured, []error) {
```

Comment:
Do you think this function would be cleaner if it didn't require both `obj` (an unstructured pod) and `pod`? We could have it return `volumeSnapshots` and call `restic.SetPodSnapshotAnnotation` outside of here.

Reply:
Yes. updated and this is now much cleaner :)